src/cryptonote_core/service_node_list.h

```diff
@@ -303,7 +303,7 @@ namespace service_nodes
       BEGIN_SERIALIZE()
         FIELD(version)
         FIELD(height)
-        FIELD_N("deregister_quorum", quorums[(size_t)quorum_type::deregister])
+        FIELD_N("uptime_quorum", quorums[(size_t)quorum_type::uptime])
```

Comment:
Is it important that this field tag stay the same for serialization for version 2?

Reply:
See version stuff, we can start fresh since we're forcing a rescan.